en/docs/next/api-portal/rest-api/schemas.md

```
@@ -1646,13 +1609,13 @@ A single delivery attempt made to a webhook subscriber.
 
 ```
 
-OAuth access token generation payload. `consumerSecret` is required — the portal uses it to call the Authorization Server token endpoint directly.
+OAuth access token generation payload. `consumerSecret` is required—the portal uses it to call the Authorization Server token endpoint directly.
 
 ### Properties
```

markdownlint-cli2 (0.23.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)